common/src/main/java/com/example/android/uamp/common/MusicServiceConnection.kt

```diff
@@ -77,8 +78,11 @@ class MusicServiceConnection(context: Context, serviceComponent: ComponentName)
         .apply { postValue(EMPTY_PLAYBACK_STATE) }
     val nowPlaying = MutableLiveData<MediaItem>()
         .apply { postValue(NOTHING_PLAYING) }
+    val isAppSpatializationEnabled = MutableLiveData<Boolean>()
```

Comment:
I think the `LiveData`s here can also use the newer constructor for initialization, i.e. `MutableLiveData<Boolean>(true)` for this line.

Reply:
That was my first approach but the following issue get's raised by the compiler:

Too many arguments for public constructor MutableLiveData<T : Any!>() defined in androidx.lifecycle.MutableLiveData